modules/manage/pages/cluster-maintenance/nodewise-partition-recovery.adoc

```diff
@@ -0,0 +1,45 @@
+= Node-wise Partition Recovery
+:description: Feature to recover dead nodes.
```

Comment:
'Feature to recover partitions that have lost a majority of replicas'

This does not recover dead nodes. It recovers the data that was partially located on the dead nodes. the nodes themselves are gone and will never be recovered, neither physically (the instance) or logically (the node ID in redpanda).